biz.aQute.repository/src/aQute/maven/api/Archive.java

```diff
@@ -11,7 +11,7 @@
 																																																								// whitespace
 			+ "(?<program>[^:]+:[^:]+)         # program\n"																																										//
 			+ "(:(?<extension>[^:]+)         # optional extension\n"																																							//
-			+ "    (:(?<classifier>[^:]+))?  # optional classifer (must be preceded by extension)\n"																															//
+			+ "    (:(?<classifier>[^:]*))?  # optional classifer (must be preceded by extension)\n"																																			//
```

Comment:
So the classifier can be empty?

Reply:
yup. The default archive actually has an empty ("") classifier.  This was supported in the API but you could not set it from a string.